src/app_service/service/ens/service.nim

```diff
@@ -0,0 +1,75 @@
+import Tables, json, chronicles, strutils
+import sets
+import options
+import chronicles, libp2p/[multihash, multicodec, cid]
+import nimcrypto
+include ../../common/json_utils
+import service_interface
+import status/statusgo_backend_new/ens as status_go
+export service_interface
+
+logScope:
+  topics = "ens-service"
+
+type 
+  Service* = ref object of ServiceInterface
+
+method delete*(self: Service) =
+  discard
+
+proc newService*(): Service =
+  result = Service()
+
+method init*(self: Service) =
+  discard
+
+method getContentHash*(self: Service, ens: string): Option[string] =
+  try:
+    let contentHash = status_go.contenthash(ens)
+    if contentHash != "":
+      return some(contentHash)
+  except Exception as e:
+    let errDescription = e.msg
+    error "error: ", errDescription
+  return none(string)
+
+method decodeENSContentHash*(self: Service, value: string): tuple[ensType: ENSType, output: string] =
+  if value == "":
+    return (ENSType.UNKNOWN, "")
+
+  if value[0..5] == "e40101":
+    return (ENSType.SWARM, value.split("1b20")[1])
+
+  if value[0..7] == "e3010170":
```

Comment:
There are a couple of string constants here. Is it possible to put them is `const`s?